### ISSUE TITLE
[astro-patser] Template directives are considered an error

### ISSUE THREAD

**Issue by kagankan** [markuplint/markuplint#575]
- Markuplint version: `2.11.0`
- Parser lang: Astro
- Node.js version: `16.16.0`
- OS: Windows

## Describe the bug

Astro provides some directives.
https://docs.astro.build/en/reference/directives-reference/

markuplint reports error for these directives.

## Code Example

```html
<main class:list={["aaa", "bbb"]} set:text="aaa"></main>

<style is:global>
  main {
    padding: 1rem;
  }
</style>
```

.markuplintrc.js:

```js
module.exports = {
  extends: ["markuplint:recommended"],
  parser: {
    ".astro$": "@markuplint/astro-parser",
  },
};
```

Output:

```
<markuplint> error: 属性「class:list」は許可されていません (invalid-attr) C:\Users\xxx\src\pages\index.astro:1:7
   1: <main•class:list={["aaa",•"bbb"]}•set:text="aaa"></main>
   2:
<markuplint> error: 属性「set:text」は許可されていません (invalid-attr) C:\Users\xxx\src\pages\index.astro:1:35
   1: <main•class:list={["aaa",•"bbb"]}•set:text="aaa"></main>
   2:
<markuplint> error: 属性「is:global」は許可されていません (invalid-attr) C:\Users\xxx\src\pages\index.astro:3:8
   2:
   3: <style•is:global>
   4: ••main•{
```

## The current behavior

Template directives are considered an error

## The expected behavior

Template directives are NOT considered an error
